src/client/components/Description Career/descriptioncareer.css

```diff
@@ -0,0 +1,39 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400&display=swap');
+
+.descriptioncareer {
+  padding: 10px;
+  padding-left: 20px;
+  background-color: #e9f4f4;
+  margin: 10px;
```

Comment:
I think, you can use a fixed height (I tried with 95%) for this class, the container to make them look equally long as design :)